controller/linodecluster_controller.go

```diff
@@ -142,10 +142,10 @@ func (r *LinodeClusterReconciler) reconcile(
 	}
 
 	// Create
-	if (clusterScope.LinodeCluster.Spec.ControlPlaneEndpoint.Host == "") || (clusterScope.LinodeCluster.Spec.Network.LoadBalancerType == "dns") {
+	if clusterScope.LinodeCluster.Spec.ControlPlaneEndpoint.Host == "" {
```

Comment:
I wonder why we needed this if loop before for dns which did reconcileCreate and why its not needed now.

Reply:
This is new functionality(we couldn't recover from the situation this is fixing before) so I would believe it was needed but we just hadn't tried this scenario yet